cfgov/unprocessed/css/pages/home.less

```diff
@@ -0,0 +1,106 @@
+.content_main__home {
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    .respond-to-max( @bp-xs-max, {
+        padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
+    } );
+}
+
+.content-l_col__home {
+    + .content-l_col__home {
+        .respond-to-max( @bp-med-max, {
+            margin-top: unit( ( @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-range( @bp-sm-min, @bp-med-max, {
+            &:before {
+                .a-rule-break();
+                content: '';
+                display: block;
+            }
+        } );
+    }
+
+    .respond-to-min( @bp-sm-min, {
+        .grid_column( 1, 1 );
```

Comment:
Responded in the original PR. The mixin assumes the second param is `12` if you only pass one, so that'd make it `1/12` instead of 12/12 like we want. I could just update it to 12, but I was worried if that ever changed what might happen.